src/seccomp/src/lib.rs

```diff
@@ -230,9 +229,11 @@
 //! [`SeccompAction`]: enum.SeccompAction.html
 //! [`SeccompFilter`]: struct.SeccompFilter.html
 //! [`action`]: struct.SeccompRule.html#action
-use std::collections::BTreeMap;
-use std::convert::TryInto;
+use bincode::Error as BincodeError;
+use serde::{Deserialize, Serialize};
+use std::collections::{BTreeMap, HashMap};
 use std::fmt::{Display, Formatter};
+use std::io::Read;
 
 /// Maximum number of instructions that a BPF program can have.
 const BPF_MAX_LEN: usize = 4096;
```

Comment:
Are we sure we do not need to update this ?

Reply:
This limit is enforced by the linux kernel